New Traffic Policy Documentation (#1011)
ngrok/ngrok-docs: docs/traffic-policy/concepts/index.mdx

---
sidebar_position: 0
title: Overview
pagination_next: traffic-policy/concepts/phases
pagination_label: Phases
---

# Traffic Policy Overview

> The ngrok Traffic Policy engine enables you to define how traffic is managed consistently across your endpoints. It allows you to inspect, manipulate, and manage traffic in a structured way.

Reviewer: ```suggestion
The ngrok Traffic Policy engine enables you to manage traffic consistently across your endpoints. It allows you to inspect, manipulate, and route traffic in a structured way.
```
